modules/formulize/include/entriesdisplay.php

```diff
@@ -1151,109 +1153,14 @@ function drawInterface($settings, $fid, $frid, $groups, $mid, $gperm_handler, $l
 			print "</table></div>";
 		}
 
-		print "<table cellpadding=10><tr><td id='titleTable' style=\"vertical-align: top;\" width=100%>";
```

Comment:
There should be a new file corresponding to these deleted lines. These lines were moved to be the "default template." I didn't see a new file added as part of this branch, so that needs to be done. Also, whatever the actual default template is in the UCOSP branch, the content of that file should now be all these lines removed from the current github copy of entriesdisplay.php, because it is the current functionality that we want to use as the current default template behaviour.

Reply:
TBD